src/libopensc/card-itacns.c

```diff
@@ -259,6 +273,14 @@ static int itacns_restore_security_env(sc_card_t *card, int se_num)
 	 * even though we know it will not be used (we don't even check it).
 	 */
 
+	if (card->type == SC_CARD_TYPE_ITACNS_CNS_IDEMIA_2021) {
+		sc_path_t path;
+		sc_format_path("3F0014009002", &path);
```

Comment:
please add a comment or a good name for this magic path

Reply:
Ok, done... check if that's enough.